service/fck-nat.sh

```diff
@@ -70,4 +70,10 @@ iptables -t nat -F
 echo "Adding NAT rule..."
 iptables -t nat -A POSTROUTING -o "$nat_interface" -j MASQUERADE -m comment --comment "NAT routing rule installed by fck-nat"
 
+if test -n "$cwagent_enabled" && test -n "$cwagent_cfg_param_name"; then
+    echo "Found cwagent_enabled and cwagent_cfg_param_arn configuration, starting CloudWatch agent..."
```

Comment:
Missed an instance of `cwagent_cfg_param_arn`

Reply:
Fixed, there were an additional couple instances that needed to be changed. Thanks!

Reply:
`English is hard`

Preach.